contracts/Vault.vy

```diff
@@ -207,6 +247,13 @@ def __init__(
             convert(self, bytes32)
         )
     )
+    log SetName(self.name)
+    log SetSymbol(self.symbol)
+    log SetRewards(self.rewards)
+    log SetGuardian(self.guardian)
+    log SetPerformanceFee(self.performanceFee)
+    log SetManagementFee(self.managementFee)
+    log SetDepositLimit(self.depositLimit)
```

Comment:
Use the input values/literals/constants for these, saving like ~1950 gas each or so

Also, put each one after where it's set

Reply:
cool -- didn't know this re. gas